src/test/resources/2321.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This suite file is not being referred to in the main `src/test/resources/testng.xml` file